tests/data/env_metadata/envpy37win/Lib/site-packages/pywin32-223-py3.7-win-amd64.egg/EGG-INFO/scripts/pywin32_postinstall.py

```diff
@@ -458,7 +456,7 @@ def install():
     # importing win32com.client ensures the gen_py dir created - not strictly
     # necessary to do now, but this makes the installation "complete"
     try:
-        import win32com.client
+        pass
     except ImportError:
         # Don't let this error sound fatal
         pass
```

Comment:
remove

Reply:
I'll revert this one, it should be ignored